packages/synapse-react-client/src/utils/hooks/useDetectSSOCode.test.ts

```diff
@@ -46,16 +47,30 @@ const mockBindOAuthProviderToAccount = jest.spyOn(
   'bindOAuthProviderToAccount',
 )
 
+const UNAUTHENTICATED_CONTEXT: SynapseContextType = {
+  ...MOCK_CONTEXT_VALUE,
+  accessToken: undefined,
+}
+
 describe('useDetectSSOCode tests', () => {
+  function renderHook(
+    options: UseDetectSSOCodeOptions,
+    contextOverrides?: Partial<SynapseContextType>,
+  ) {
+    return _renderHook(() => useDetectSSOCode(options), {
+      wrapper: createWrapper(contextOverrides),
+    })
```

Comment:
This hook was recently changed and it now uses `useSynapseContext`. Instead of mocking `useSynapseContext` in just a few key tests, always use the real thing and contextually pass a fake context object depending on the test.